kpops/api/__init__.py

```diff
@@ -217,7 +217,6 @@ def clean(
     )
 
     async def clean_runner(component: PipelineComponent):
-        await component.destroy(dry_run)
```

Comment:
TODO: This is problematic. We should fetch the Helm values first and then destroy the app.